CHANGELOG.md

```diff
@@ -93,6 +98,10 @@ and this project adheres to
   the `start_time_us` and `start_time_cpu_us` metrics. These parameters were
   never meant to be used by end customers, and we recommend doing any such time
   adjustments outside Firecracker.
+- Booting with microVM kernels that rely on MPTable is deprecated and support
```

Comment:
I would add a note that this is an x86 only deprecation just in case.

Reply:
agreed, I will add it.